### PR TITLE
Update player with map creative button 

### PR DESCRIPTION
**Pull request by kristen-m**
--The allow location access button is only displayed if the map creative is selected

### CODE REVIEW

File: src/main/webapp/player/simid_player.html [hunk 1]
```diff
@@ -61,6 +61,9 @@
           ];
           adParams = JSON.stringify(adParamsObj);
           break;
+        case 'maps/simid_map.html':
```
Comment (damargulis) [inline src/main/webapp/player/simid_player.html:64]: If you select simid_map, but then select something else again, doesn't this button stay visible?

Comment (kristen-m) [reply src/main/webapp/player/simid_player.html:65]: whoops, yep. Fixed that

File: src/main/webapp/player/simid_player.html [hunk 2]
```diff
@@ -61,6 +62,9 @@
...
+          document.getElementById('geolocation').style.visibility = 'visible';
+          break;
```
Comment (ryanthompson591) [inline src/main/webapp/player/simid_player.html:67]: other code changes adParams here. Do you need to do that. If not consider emptying that field.

File: src/main/webapp/player/simid_player.html [hunk 3]
```diff
@@ -46,6 +46,10 @@
     window.switchCreative = (value) => {
       document.getElementById('creative_url').value = '../creatives/' + value;
       // populate ad parameters with some hard coded values
+      document.getElementById('geolocation').style.visibility = 'hidden';
+      if (value == 'maps/simid_map.html') {
```
Comment (damargulis) [inline src/main/webapp/player/simid_player.html:50]: Keep this as part of the switch, no need to have an extra if statement if you're gonna use a switch anyway.

Comment (kristen-m) [reply src/main/webapp/player/simid_player.html]: Fixed

File: src/main/webapp/player/simid_player.html (hunk 2 above)
Comment (kristen-m) [reply src/main/webapp/player/simid_player.html:74]: fixed

**Pull request merged**